src/app/meteo/form-filter/switcher/switcher.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, Input, forwardRef } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+
+const CUSTOM_CHECKBOX_VALUE_ACCESSOR = {
+    provide: NG_VALUE_ACCESSOR,
+    useExisting: forwardRef(() => SwitcherComponent),
+    multi: true
+};
+
+@Component({
+    selector: 'wapi-checkbox',
+    templateUrl: 'switcher.component.html',
+    styleUrls: ['switcher.component.css'],
+    providers: [CUSTOM_CHECKBOX_VALUE_ACCESSOR]
+})
+
+export class SwitcherComponent implements ControlValueAccessor {
+    @Input('value') val: boolean = false;
```

Comment:
@LeraKruchek you mean @Input or value. It the first one - sure, the 'value' - no.